docs/a320-simvars.md

```diff
@@ -1431,6 +1431,7 @@ In the variables below, {number} should be replaced with one item in the set: {
     - 0 is CAPT, 1 is NORM, 2 is F/O
 
 - A32NX_ADIRS_ADIRU_{number}_STATE
+    - TODO delete
```

Comment:
I would here indicate what to use instead, and use wording like "deprecated"

Reply:
Ah, good catch. I had intended to delete these simvars as part of this PR when I wrote those comments, so they were for me only. I will either do that (which will require changes in other systems), or most likely deprecate as you suggest...